.goreleaser.yml

```diff
@@ -8,6 +8,7 @@ before:
 builds:
   - env:
       - CGO_ENABLED=0
+    binary: rainforest
```

Comment:
@jbarber @magni- @AJFunk this is the line that should fix it, the rest is just testing. Locally when I build, it a) works b) gives me the correct binary name c) that binary runs:

```bash
russ@Russells-iMac-Pro rainforest-cli % goreleaser build --snapshot
   • building...      
   • loading config file       file=.goreleaser.yml
   • loading environment variables
   • getting and validating git state
      • building...               commit=3bc8022ec4e031b613b8ec3f0753e5caac1c1e88 latest tag=v2.22.2
      • pipe skipped              error=disabled during snapshot mode
   • parsing tag      
   • running before hooks
      • running                   hook=go mod tidy
      • running                   hook=go generate ./...
   • setting defaults 
      • snapshotting     
      • github/gitlab/gitea releases
      • project name     
      • loading go mod information
      • building binaries
      • creating source archive
      • archives         
      • linux packages   
      • snapcraft packages
      • calculating checksums
      • signing artifacts
      • signing docker images
      • docker images    
      • docker manifests 
      • artifactory      
      • blobs            
      • homebrew tap formula
      • scoop manifests  
      • discord          
      • reddit           
      • slack            
      • teams            
      • twitter          
      • smtp             
      • milestones       
   • snapshotting     
      • building snapshot...      version=v2.22.2-next
   • checking ./dist  
   • loading go mod information
   • writing effective config file
      • writing                   config=dist/config.yaml
   • building binaries
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_darwin_arm64/rainforest
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_linux_arm64/rainforest
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_linux_amd64/rainforest
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_windows_arm64/rainforest.exe
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_linux_386/rainforest
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_darwin_amd64/rainforest
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_windows_386/rainforest.exe
      • building                  binary=/Users/russ/Sites/rf/rainforest-cli/dist/rainforest-cli_windows_amd64/rainforest.exe
   • build succeeded after 2.64s
russ@Russells-iMac-Pro rainforest-cli % dist/rainforest-cli_darwin_amd64/rainforest --skip-update -v
Rainforest CLI version 2.22.2
```